DATA_SOURCES.md

```diff
@@ -321,6 +321,7 @@ For many European countries, data is available from [ENTSO-E](https://transparen
 - Puerto Rico: [EIA](https://www.eia.gov/electricity/data/eia860M/)
 - Qatar: [IRENA](https://www.irena.org/publications/2022/Apr/Renewable-Capacity-Statistics-2022)
 - Romania:
+  - Latest?: [ANRE](https://www.anre.ro/ro/energie-electrica/rapoarte/puterea-instalata-in-capacitatiile-de-productie-energie-electrica)
   - Nuclear: [Nuclearelectrica](http://www.nuclearelectrica.ro/cne/)
   - Other: [ENTSO-E](https://transparency.entsoe.eu/generation/r2/installedGenerationCapacityAggregation/show)
```

Comment:
Nice, if this contains all capacities I think we can get rid of the two other sources:
```suggestion
- Romania: [ANRE](https://www.anre.ro/ro/energie-electrica/rapoarte/puterea-instalata-in-capacitatiile-de-productie-energie-electrica)
```

Reply:
Left ENTSO-E just in case ANRE goes offline 😄